VCTwoFluidStaggeredStokesBoxRelaxationFACOperator.h

```diff
@@ -196,6 +197,7 @@ class VCTwoFluidStaggeredStokesBoxRelaxationFACOperator : public FACPrecondition
 protected:
     int d_thn_idx = IBTK::invalid_index;
     int d_un_scr_idx = IBTK::invalid_index, d_us_scr_idx = IBTK::invalid_index, d_p_scr_idx = IBTK::invalid_index;
+    double d_w = IBTK::invalid_index; // under relaxation factor
```

Comment:
```suggestion
    double d_w = std::numeric_limits<double>::quiet_NaN(); // under relaxation factor
```
`IBTK::invalid_index` is an integer (and d_w is not an index).